test/e2e/components/vl-map-layer.js

```diff
@@ -9,6 +9,10 @@ class VlMapLayer extends VlElement {
     return this.getAttribute('name');
   }
 
+  async getNumberOfFeatures() {
+    return this.driver.executeScript(`return arguments[0].layer.getSource().getFeatures().length;`, this);
```

Comment:
misschien hier `(await getFeatures()).length` doen?

Reply:
ik had daar miserie mee (maar ik weet niet of het exact hetzelfde was) dat er iets stale was geworden net tussen waar ge dan de features vroeg en dan daar de length van

Reply:
ja deze had ik overgenomen uit de branch van Pascal, zal het eens uittesten

Reply:
lijkt geen problemen te geven, ik pas het aan